src/components/Search.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+//redux
+import { useDispatch } from "react-redux";
+import { loadWeatherData } from "../actions/weatherAction";
+
+const Search = () => {
+  const [input, setInput] = useState("");
+  const dispatch = useDispatch();
+
+  //event listeners
+  const handleInput = (e) => {
+    setInput(e.target.value);
+  };
+
+  const handleSearchButton = () => {
```

Comment:
This is being nitpicky, but I like to name these handlers with the action that's triggering them - so `handlSearchButtonClick` or `handleInputChange`